packages/mui-material/src/Tabs/Tabs.test.js

```diff
@@ -1397,4 +1334,87 @@ describe('<Tabs />', () => {
       ]);
     });
   });
+
+  describe('dynamic tabs', () => {
```

Comment:
@SaidMarar These tests pass in master HEAD branch, which means that it is not testing or covering the bugs. I think you need to replicate the behaviour of the [before CodeSandbox](https://codesandbox.io/s/interesting-before-b3kdvn?file=/src/App.js) in the tests by displaying tabs vertically, switching to the last tab and then ensuring that deleting it does not cause the below scroll button to appear.

Reply:
@ZeeshanTamboli the [before CodeSandbox](https://codesandbox.io/s/interesting-before-b3kdvn?file=/src/App.js) is for the version **5.11.7**.  But in the latest version it has different behavior the scrollbutton is blinking then disappear so fast, please take a look at the [codeSanbox](https://codesandbox.io/s/pedantic-bush-vq9ltt?file=/Demo.tsx), I think that's why the test is not failing in master but the bug is there. 

![ezgif com-video-to-gif](https://github.com/mui/material-ui/assets/20561190/2b7357b0-20d7-4c1f-bdcc-44a522bc3481)